docs/data/charts/highlighting/ControlledAxisHighlight.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import Stack from '@mui/material/Stack';
+import Box from '@mui/material/Box';
+import ToggleButtonGroup from '@mui/material/ToggleButtonGroup';
+import ToggleButton from '@mui/material/ToggleButton';
+import { BarChart, BarChartProps } from '@mui/x-charts/BarChart';
+import FormControl from '@mui/material/FormControl';
+import FormControlLabel from '@mui/material/FormControlLabel';
+import FormLabel from '@mui/material/FormLabel';
+import Radio from '@mui/material/Radio';
+import RadioGroup from '@mui/material/RadioGroup';
+import { AxisItemIdentifier } from '@mui/x-charts/models';
+import { LineChart, LineChartProps } from '@mui/x-charts/LineChart';
+
+export default function ControlledAxisHighlight() {
```

Comment:
Should we render both the bar and line charts simultaneously to show both controlled and synchronized axis highlighting? I suspect synchronized axis highlighting is something that users will want, so we'd show how to do that.